build.gradle

```diff
@@ -333,7 +330,7 @@ def getGitVersion() {
     describeStdOut.toString().substring(1).trim()
 }
 
-apply from: 'gradle/javaToolchain.gradle'
 apply from: 'gradle/publish.gradle'
 apply from: 'gradle/deploy.gradle'
 apply from: 'gradle/javadoc.gradle'
+apply from: 'gradle/javaToolchain.gradle'
```

Comment:
There's a significance to the order?  Another thing I didn't know...

Reply:
I just changed it as part of debugging and getting everything done - it might not actually matter.

Reply:
Reverted this change back